src/SqlPersistence/Config/SqlDialect_Oracle.cs

```diff
@@ -12,6 +13,8 @@ public abstract partial class SqlDialect
         /// </summary>
         public partial class Oracle : SqlDialect
         {
+            volatile PropertyInfo bindByNameProperty;
```

Comment:
i though volatile was bad for perf on the hot path https://blogs.msdn.microsoft.com/brada/2004/05/13/volatile-and-memorybarrier/